src/commands/transaction/transfer.ts

```diff
@@ -159,6 +164,12 @@ export default class extends AnnounceTransactionsCommand {
             options.maxFee ? UInt64.fromNumericString(options.maxFee) : UInt64.fromUint(0));
 
         const signedTransaction = profile.account.sign(transferTransaction, profile.networkGenerationHash);
-        this.announceTransaction(signedTransaction, profile.url);
+
+        const payload = signedTransaction.payload;
+        const shortPayload = this.shortPayload(payload);
+        const isAnnounce = readlineSync.keyInYN('Do you want to announce this transaction? Payload: ' + shortPayload);
+        if (isAnnounce) {
+            this.announceTransaction(signedTransaction, profile.url);
+        }
```

Comment:
what happens to the transaction in case it is not announced ? I would expect the CLI to print the payload then at least ?